.git-blame-ignore-revs

```diff
@@ -1,2 +1,5 @@
 # Invoking prettier for the first time
 b16d4388c0457cd6fe02fc09947712d6f3afed7c
+
+# Invoking prettier v 3.2.4
+ba32601a823ac6e03673b52ab0956b99cfb9f5a1
```

Comment:
Ah-hah! So this manifest ensures `git blame` ignores this commit when blaming line changes? Very cool!

Reply:
Yeah!  h/t @jasonaowen for this pattern